src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
-	option: [ 1, "<select multiple='multiple'>", "</select>" ],
-
 	// XHTML parsers do not magically insert elements in the
 	// same way that tag soup parsers do. So we cannot shorten
 	// this by omitting <tbody> or other required elements.
+	option: [ 1, "<select multiple='multiple'>", "</select>" ],
+
```

Comment:
I'm not sure why you would like to move code around, that just creates trash in the blame